python/paddle/tensor/__init__.py

```diff
@@ -112,6 +112,9 @@
 from .logic import isclose  # noqa: F401
 from .logic import equal_all  # noqa: F401
 from .logic import is_tensor  # noqa: F401
+from .manipulation import atleast_1d  # noqa: F401
+from .manipulation import atleast_2d  # noqa: F401
+from .manipulation import atleast_3d  # noqa: F401
```

Comment:
Shall we need to support the usage of `x.atleast_1d(), x.atleast_2d(), x.atleast_3d()` when `x` is a tensor ? If so, add them to the list of `tensor_method_func` below